leshan-core/src/test/java/org/eclipse/leshan/core/util/TimestampUtilTest.java

```diff
@@ -0,0 +1,72 @@
+package org.eclipse.leshan.core.util;
```

Comment:
It misses header in this file.
This is maybe something we should check at build time, I don't know if this is possible :thinking: